Remove gulp-es6-module-transpiler, use Babel?

I've forked `metrics-graphics` and I'm trying to build. Gulp returns this error: `Error: Cannot find module 'es6-module-transpiler/node_modules/recast'`.

I took a look in the `gulp-es6-module-transpiler` dependency - they have very bad code (https://github.com/ryanseddon/gulp-es6-module-transpiler/blob/master/lib/index.js#L13).

I don't really know what's wrong and I don't even understand how this module's being used in the build process since it's not referenced after being required, but I tried removing the require and that broke other things. Any chance Babel could be used instead?

--- a/gulp/index.js
+++ b/gulp/index.js
@@ -10,8 +10,7 @@ var
   //minifycss = require('gulp-minify-css'), // for minifiing css
   jshint = require('gulp-jshint'),
   testem = require('gulp-testem'),
-  connect = require('gulp-connect'),
-  es6ModuleTranspiler = require("gulp-es6-module-transpiler");
+  connect = require('gulp-connect');
 
 // paths
 var

--- a/package.json
+++ b/package.json
@@ -43,7 +43,6 @@
     "gulp": "^3.8.10",
     "gulp-concat": "^2.4.2",
     "gulp-connect": "^2.2.0",
-    "gulp-es6-module-transpiler": "^0.2.0",
     "gulp-jshint": "^1.9.0",
     "gulp-rename": "^1.2.0",
     "gulp-rimraf": "^0.1.1",

